core/includes/unicode.inc

```diff
@@ -323,10 +323,11 @@ function backdrop_truncate_bytes($string, $len) {
  * @return string
  *   The truncated string.
  */
-function truncate_utf8($string, $max_length, $wordsafe = FALSE, $add_ellipsis = FALSE, $min_wordsafe_length = 1) {
+function truncate_utf8($string, $max_length, $wordsafe = FALSE, $add_ellipsis = FALSE, $min_wordsafe_length = 1, $include_newlines = FALSE) {
```

Comment:
We'll need an `@since` line here.

Reply:
Yep, was rushed last night and lay in bed realising I'd forgotten the docblock...